airconsole-1.9.0-alpha.js

```diff
@@ -1092,11 +1131,63 @@ AirConsole.prototype.init_ = function(opts) {
     version: me.version,
     device_motion: opts.device_motion,
     synchronize_time: opts.synchronize_time,
+    silence_players: me.silence_players,
     location: me.getLocationUrl_(),
     translation: opts.translation
   });
+  this.enablePlayerSilencing(me.silence_players);
```

Comment:
should only be called if it's activated. additional uneeded device state is not a good idea